app/containers/message/User.js

```diff
@@ -70,6 +71,7 @@ export default class User extends React.PureComponent {
 						{aliasUsername}
 					</Text>
 				</View>
+				{edited ? <Text style={styles.edited}> (edited)</Text> : null}
```

Comment:
it would be better if you render 'edited' message not in 'User' component in a separate functional component e.g. 'EditedTag' because it's confusing now.